### DISCUSSION TITLE
Open a tab in a new window using middle click

### DISCUSSION THREAD

**Discussion by Omonoboko** [piroor/treestyletab#3159] [Q&A]
I would like to be able to open a tab in a new window using middle click. How do i go about this?

**Comment by irvinm** [comment]
I am assuming you mean that middle clicking-ing on a link to open a new window ...

If so, this should help you (not TST related):  https://support.mozilla.org/en-US/questions/1289787

**Reply by Omonoboko** [reply]
Thanks for your help, but i want to be able to open an existing tab into a new window using middleclick. Middling clicking on a TST tab right now straight up deletes it. Is there any way i can change that function?

**Comment by piroor** [comment]
TST doesn't have this feature and I won't add it because it is out of scope of this project. Instead it needs to be done with a helper addon. There is the [list of known helper addons](https://github.com/piroor/treestyletab#extensions-that-extend-tst) and you may have to implement it by yourself if there is no existing helper addon providing the behavior.
